src/app/application.cpp

```diff
@@ -171,6 +171,11 @@ Application::Application(const QString &id, int &argc, char **argv)
         m_fileLogger = new FileLogger(fileLoggerPath(), isFileLoggerBackup(), fileLoggerMaxSize(), isFileLoggerDeleteOld(), fileLoggerAge(), static_cast<FileLogger::FileLogAgeType>(fileLoggerAgeType()));
 
     Logger::instance()->addMessage(tr("qBittorrent %1 started", "qBittorrent v3.2.0alpha started").arg(QBT_VERSION));
+    Logger::instance()->addMessage(tr("Using config directory: %1").arg(Profile::instance().location(SpecialFolder::Config)));
+    if (portableModeEnabled)
+        Logger::instance()->addMessage(tr("Running in portable mode (auto detected)"));
```

Comment:
will this work?
`tr("Running in portable mode. Auto detected profile folder at: %1").arg(profileDir)`

Reply:
It will show almost the same path as message above. Are you sure we need both of them?

Reply:
>Are you sure we need both of them?

I don't insist on that. But the point is `(auto detected)` has not enough information to users, they won't know what path is auto detected.

Reply:
Or maybe:
```c++
if (portableModeEnabled) {
    // ...
}
else {
    Logger::instance()->addMessage(tr("Using config directory: %1").arg(Profile::instance().location(SpecialFolder::Config)));
}
```